src/components/FileRepo/arrangerStyle.js

```diff
@@ -198,7 +198,6 @@ export default () => css`
     cursor: pointer;
     display: flex;
     align-items: center;
-    justify-content: flex-end;
```

Comment:
@jberube looking great so far. Is this affecting CohortBuilder styles? Might be better to change a `CohortBuilder` style than to change `FileRepo` style?

Reply:
The same style is inherited by the only component that uses it.
As a rule of thumb, I suggest that everything that concerns the positioning of a component should be kept outside of that component's styles. Not all usages of a component are aligned the same way.

Reply:
totally agree @jberube, is this affecting the new feature though? I am just a bit hesitant of changes to existing working features right now. If this is affecting new features, my bigger concern is the fact that this style here is not scoped to the FileRepo only, but addressing that is probably a bigger task.

Reply:
I've checked the FileRepo afterward, and it didn't changed there, since it is also set this way:
```
.aggregation-card .showMore-wrapper {
    display: -ms-flexbox;
    display: flex;
    -ms-flex-pack: end;
    justify-content: flex-end;
}
```